python/lsst/ip/diffim/imageMapReduce.py

```diff
@@ -261,6 +261,7 @@ def run(self, mapperResults, exposure, **kwargs):
             patchMI = item.getMaskedImage()
             isNotNan = ~(np.isnan(patchMI.getImage().getArray()) |
                          np.isnan(patchMI.getVariance().getArray()))
```

Comment:
At risk of premature optimization, I will implement everything you've recommended here.